composer.json

```diff
@@ -15,16 +15,17 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "symfony/event-dispatcher": "^2.2|^3.0|^4.0",
-        "symfony/options-resolver": "^2.6|^3.0|^4.0",
-        "fsi/data-indexer" : "0.9.*"
+        "php": "^7.1|^8.0",
+        "symfony/event-dispatcher": "^3.4|^4.0",
```

Comment:
Did not think of this before, but shouldn't we require Symfony components in versions `4.4.x` as well? Less dependency resolving and we don't use lower minor versions anyway.

Reply:
maybe in next round of updates